src/Jade/Compiler.php

```diff
@@ -708,7 +714,7 @@ protected function createStatements()
         if (count($statements) == 0) {
             $php_string = call_user_func_array('sprintf', $code_format);
 
-            return '<?php ' . $php_string . ' ?>';
+            return '<?php ' . $php_string . " {$this->closingTag}";
```

Comment:
Okay, I agree with you, just didn't realized what I was doing. I'll change it ASAP.